dependencies.gradle

```diff
@@ -2,16 +2,16 @@ ext {
     libraries = [
             app: [
                     annotations     : 'org.jetbrains:annotations:13.0',
-                    appCompat       : 'com.android.support:appcompat-v7:25.0.0',
+                    appCompat       : 'com.android.support:appcompat-v7:25.0.1',
                     crashlytics     : 'com.crashlytics.sdk.android:crashlytics:2.6.5',
                     gson            : 'com.google.code.gson:gson:2.8.0',
                     materialTabStrip: 'com.jpardogo.materialtabstrip:library:1.0.8',
                     parallaxScroll  : 'com.github.nirhart:parallaxscroll:1.0',
-                    playAnalytics   : 'com.google.android.gms:play-services-analytics:9.8.0',
-                    playMaps        : 'com.google.android.gms:play-services-maps:9.8.0',
+                    playAnalytics   : 'com.google.android.gms:play-services-analytics:10.0.0',
+                    playMaps        : 'com.google.android.gms:play-services-maps:10.0.0',
```

Comment:
Worth extracting this version number to a constant, so that it's always in sync between different play services libraries?